src/main/java/io/kabanero/api/restricted/CollectionsEndpoints.java

```diff
@@ -116,7 +118,7 @@ public Response listCollections(@CookieParam(JWT_COOKIE_KEY) String jwt) throws
     @GET
     @Path("/version")
     @Produces(MediaType.APPLICATION_JSON)
-    public Response cliVersion(@CookieParam(JWT_COOKIE_KEY) String jwt) throws ClientProtocolException, IOException {
+    public Response cliVersion(@CookieParam(JWT_COOKIE_KEY) String jwt) throws ClientProtocolException, IOException, ApiException, GeneralSecurityException {
```

Comment:
doesn't look like you touched the method body, how come you added 2 new exceptions to the method signature? Same for listCollections endpoint abvoe

Reply:
They both call `setCLIURL` and `setCLIURL` calls `getAnInstance` in the Kabanero Client.  That method calls `KabaneroClient.getApiClient()` which throws the `GeneralSecurityException`.  It also calls the KabaneroApi's `getKabanero` method which throws the `ApiException`